Common/tests/Pollster/PollsterTest.cs

```diff
@@ -441,7 +440,8 @@ public WaitAsyncPipe(double delay)
 
     public async Task<ProcessReply> ReadAsync(CancellationToken cancellationToken)
     {
-      await Task.Delay(TimeSpan.FromMilliseconds(delay_))
+      await Task.Delay(TimeSpan.FromMilliseconds(delay_),
+                       cancellationToken)
```

Comment:
I've checked quickly, and it seems this change is the cause of the failing test. We can retain the old behavior without the warning like this:
```suggestion
                       CancellationToken.None)
```